src/assets/src/components/queue.tsx

```diff
@@ -37,7 +37,7 @@ const JoinQueue: React.FC<JoinQueueProps> = (props) => {
         <>
         <Row>
             <Col lg>
-                <p className="mb-0">Select Meeting Type<span className="required">*</span></p>
+                <p className="mb-0">Select Meeting Type <span className="required">*</span></p>
```

Comment:
I know this is just adding a space here but it looks like using the HTML form validation is recommended over an * both for usage and accessibility. 

https://www.accessibility-developer-guide.com/examples/forms/required/#using-html-5-client-side-validations
https://react-bootstrap.netlify.app/docs/forms/validation

Was this not changed just because that's a separate (more effort) issue? We should probably file it as a new issue to fix it in the future.

Reply:
I think there are some interesting ideas in this article. I'm not exactly sure how it interacts with dropdowns, or this particular case. Maybe the required indicator isn't the best here anyway. I'm going to open this as a separate issue to reconsider, since it would affect other areas of the application.

Reply:
See #437.